cmd/kms-rest/startcmd/start.go

```diff
@@ -164,13 +164,14 @@ func startKmsService(parameters *kmsRestParameters, srv server) error {
 		return nil
 	}
 
+	logger := log.New("hub-kms/restapi")
```

Comment:
Probably want this as a global variable so other method can access `logger` if/when you add logging there... take a look at the other trustbloc projects to see how the logger is instantialized there. It would also be good to match those for consistency reasons...

(note that if the linter complains, you'll have to add an exception - you can also find the logger exception in those projects in `.golangci.yml`)

Reply:
I don't like the idea of the logger defined as a global variable. I was considering passing it as a dependency to the controller, operation, etc. I was trying to use `log.Logger` interface as a contract and `log.Log` as an implementation (both from `edge-core`) but for some reason - it didn't work - so, I abandoned that idea. But, probably, I will come back to it in my next PR. My main motivation is to be able to unit test code that uses logging.

Reply:
I believe there are some issues with trying to pass it as a dependency. I have a way to unit test code with the logger in it - I can show you.

We can discuss the logger thing separately for now just leave this PR as is